demos/android/MASVS-PLATFORM/MASTG-DEMO-0031/evaluate.py

```diff
@@ -0,0 +1,32 @@
+import re
```

Comment:
Is there a reason to have this separate script? Vs putting this logic directly in the frida script?